pkgs/applications/misc/collision/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib
+, fetchFromGitHub
+, crystal
+, libadwaita
+, gtk3
+, gobject-introspection
+, openssl
+, libxml2
+, pkg-config
+, wrapGAppsHook4
+}:
+let
+  gi_crystal = crystal.buildCrystalPackage {
```

Comment:
Why not move that into gi-crystal/default.nix?

Reply:
Good suggestion, fixed in https://github.com/NixOS/nixpkgs/pull/242681/commits/9c1cd513a09b5380ac72043e690390c54590a6bb